eZ/Publish/Core/Persistence/Legacy/Content/Type/Gateway/DoctrineDatabase.php

```diff
@@ -158,7 +158,7 @@ public function insertGroup(Group $group)
         );
         $q->prepare()->execute();
 
-        return $this->dbHandler->lastInsertId(
+        return (int)$this->dbHandler->lastInsertId(
```

Comment:
If we add `(int)` here than we should change also `@return mixed Group ID`  https://github.com/ezsystems/ezpublish-kernel/pull/2856/files#diff-f7709a1560b54a55dbc2fd87a1ee5bd9R133